libraries/botbuilder-dialogs-declarative/tests/schemaMergeTest.js

```diff
@@ -56,21 +45,20 @@ describe('Schema Merge Tests', function () {
     // This fixture creates or updates test.schema by calling bf dialog:merge on all the schema files in the solution.
     // This will install the latest version of botframework-cli if the schema changed and npm is present.
     // This only runs on Windows platforms.
-    maybeIt('should merge dialog schemas', async function () {
+    it('should merge dialog schemas', async function () {
         this.timeout(150000);
         fs.unlinkSync(testsSchemaPath);
 
         // Merge all schema files.
-        const sdkRoot = path.join(__dirname, '..', '..', '..');
         const mergeCommand = [
-            `bf dialog:merge "${sdkRoot}/libraries/**/*.schema"`,
-            `"${sdkRoot}/libraries/**/*.uischema"`,
-            '!**/testbot.schema',
-            '!**/botbuilder-dialogs-adaptive/tests/schema/*.*',
+            'bf dialog:merge ./libraries/**/*.schema',
+            './libraries/**/*.uischema',
+            '"!"**/testbot.schema',
+            '"!"**/botbuilder-dialogs-adaptive/tests/schema/*.*',
             `-o "${testsSchemaPath}"`,
         ];
         try {
-            await runCommand(['npx -p @microsoft/botframework-cli@latest', ...mergeCommand].join(' '));
+            await runCommand(mergeCommand.join(' '));
```

Comment:
Note: I'm not sure how I missed this in previous PR. `npx` should only be run on the second try, and does so on line 71.

Reply:
I overrode it as someone else was having trouble with this running in CI. Merged the change while you were OOF, so that's my fault.

Reply:
np. Thought I was going crazy!